pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-slack/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pidgin }:
+
+stdenv.mkDerivation {
+  pname = "purple-slack";
+  version = "unstable-2019-08-18";
```

Comment:
```suggestion
  version = "2019-08-18";
```

Reply:
Should be resolved in latest force push :)